src/app/code/community/FireGento/DynamicCategory/Model/Rule/Condition/Product.php

```diff
@@ -81,13 +81,13 @@ protected function _addSpecialAttributes(array &$attributes)
      */
     public function loadAttributeOptions()
     {
-        $productAttributes = Mage::getResourceSingleton('catalog/product')
```

Comment:
A caching issue occurred where the is_visible data wasn't set for the price attribute.